src/controllers/hello-world.ts

```diff
@@ -4,21 +4,27 @@
 // License text available at https://opensource.org/licenses/MIT
 'use strict';
 
-import { api, inject } from '@loopback/core';
-import { helloWorldApi } from '../apidefs/hello.world.api';
-import { authenticate, AuthenticationBindings, UserProfile } from '@loopback/authentication';
-import { UserRepository } from '../repositories/user.repository';
-import { User } from '../models/user';
+import {api, inject} from '@loopback/core';
+import {helloWorldApi} from '../apidefs/hello.world.api';
+import {
+  authenticate,
+  AuthenticationBindings,
+  UserProfile
+} from '@loopback/authentication';
+import {UserRepository} from '../repositories/user.repository';
+import {User} from '../models/user';
 
 @api(helloWorldApi)
 export class HelloWorldController {
-  @inject('repositories.UserRepository') repository: UserRepository
-  constructor(@inject(AuthenticationBindings.CURRENT_USER) private user: UserProfile) {}
+  constructor(
+    @inject(AuthenticationBindings.CURRENT_USER) private user: UserProfile,
+    @inject('repositories.UserRepository') private repository: UserRepository
```

Comment:
It makes me wonder - can/should we use `@repository(UserRepository)` instead of `@inject(repositories.UserRepository)`?